src/web/crate_details.rs

```diff
@@ -343,7 +344,18 @@ pub fn crate_details_handler(req: &mut Request) -> IronResult<Response> {
         )
     );
 
-    CrateDetailsPage { details }.into_response(req)
+    let inner_path = req.url.path()[3..].join("/");
```

Comment:
perhaps I'm missing a piece I don't know yet, but in which cases can we have an `inner_path` here? 

builds/source/features are handled by different handlers.